src/howitz/error_handlers.py

```diff
@@ -75,3 +77,36 @@ def handle_403(e):
     response.headers['HX-Trigger'] = 'htmx:responseError'
 
     return response, 403
+
+
+def handle_lost_connection(e):
+    if isinstance(e, BrokenPipeError):
+        current_app.logger.error("Lost connection to Zino server: %s", e)
+    else:
+        current_app.logger.error("Lost connection to Zino server: %s", e.args[0])
+
```

Comment:
Urgh, I meant: log with error in the `else`. logger.exception is fine for BrokenPipeError.